### PR TITLE
rsync: output warnings to stdout instead of using glog

### PR DESCRIPTION
**Pull request by csrwng**
Removes the use of glog for warning output

### CODE REVIEW

**Comment by csrwng** [comment]
@fabianofranz ptal

**Comment by csrwng** [comment]
or @bparees ptal

**Comment by bparees** [comment]
lgtm [merge]

**Comment by fabianofranz** [comment]
LGTM

**Comment by csrwng** [comment]
@smarterclayton or @liggitt ptal

File: pkg/cmd/cli/cmd/rsync/copymulti.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ func (ss copyStrategies) Copy(source, destination *pathSpec, out, errOut io.Writ
 		err = s.Copy(source, destination, out, errBuf)
 		if _, isSetupError := err.(strategySetupError); isSetupError {
 			glog.V(4).Infof("Error output:\n%s", errBuf.String())
-			glog.Warningf("Cannot use %s: %v", s.String(), err.Error())
+			fmt.Fprintf(out, "WARNING: cannot use %s: %v", s.String(), err.Error())
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/rsync/copymulti.go]: Should be to stderr - is that the case?

File: pkg/cmd/cli/cmd/rsync/rsync.go [hunk 2]
```diff
@@ -98,12 +97,12 @@ func NewCmdRsync(name, parent string, f *clientcmd.Factory, out, errOut io.Write
 	return cmd
 }
 
-func warnNoRsync() {
+func warnNoRsync(out io.Writer) {
 	if isWindows() {
-		glog.Warningf(noRsyncWindowsWarning)
+		fmt.Fprintf(out, noRsyncWindowsWarning)
```
Comment (smarterclayton) [inline pkg/cmd/cli/cmd/rsync/rsync.go]: Stderr

**Comment by csrwng** [comment]
@smarterclayton thx, comments addressed

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**